apps/portal-front/src/components/enroll/state/result.tsx

```diff
@@ -38,15 +38,16 @@ export const EnrollStateResult: React.FC<Props> = ({
     );
 
   useMountingLoader(loadQuery, { organizationId: organizationId });
-
   if (!organizationId) {
     return null;
   }
 
   if (
     !queryRef ||
     !canEnrollState ||
-    (canEnrollState.status === 'never_enrolled' && enrollmentStatus === 'idle')
+    (canEnrollState.status === 'never_enrolled' &&
```

Comment:
Please create ENUMs in GraphQL for this statuses.
They will be available in TS enums in back and front as well